catalog/core/catalog-core-impl/filter-proxy/src/main/java/ddf/catalog/filter/proxy/adapter/GeotoolsFilterAdapterImpl.java

```diff
@@ -949,7 +976,10 @@ private ExpressionValues getExpressions(Expression expression1, Expression expre
         String propertyName;
         Object literal;
 
-        if (expression1 instanceof PropertyName && expression2 instanceof Literal) {
+        if (expression1 instanceof Function && expression2 instanceof Literal) {
+            return new ExpressionValues((ProximityArguments) expression1.accept(this, delegate),
```

Comment:
This currently doesn't mess up future extensions because this is the first function to be added to `getExpressions`, but it should be more granular.  I've updated this conditional to specifically check for ProximityFunctions